src/components/Time.jsx

```diff
@@ -0,0 +1,39 @@
+import PropTypes from "prop-types";
+
+export const Time = (props) => {
+  const now = new Date();
```

Comment:
Good job with the logic for calculating the time stam; I used another approach with the Dependency of "moment". Maybe you find it useful next time 😀